tests/vm_test.go

```diff
@@ -274,21 +274,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		)
 
 		It("[test_id:3161]should carry annotations to VMI", func() {
```

Comment:
aint we update annotations after creation so we will see it still propogate from VM to vmi ?
(versus adding it before the creation)

Reply:
This is not relevant for the test.

Reply:
Thanks @xpivarc :)